web/src/components/LanguageFailure.tsx

```diff
@@ -13,31 +13,25 @@ const ChooseLanguage = styled.p`
 `
 
 type PropsType = {
-  cities: Array<CityModel>
-  cityCode: string
-  pathname: string
+  cityModel: CityModel
   languageCode: string
   languageChangePaths: Array<{ code: string; path: string | null; name: string }>
-  t: TFunction
 }
 
-export class LanguageFailure extends React.PureComponent<PropsType> {
-  render(): ReactNode {
-    const { t, cities, languageChangePaths, cityCode, pathname, languageCode } = this.props
-    const title = CityModel.findCityName(cities, cityCode)
-    return (
-      <>
-        {title && <Caption title={title} />}
-        <ChooseLanguage>{`${t('notFound.language')} ${t('chooseALanguage')}`}</ChooseLanguage>
-        <LanguageSelector
-          isHeaderActionItem={false}
-          pathname={pathname}
-          languageCode={languageCode}
-          languageChangePaths={languageChangePaths}
-        />
-      </>
-    )
-  }
+const LanguageFailure = ({ cityModel, languageCode, languageChangePaths }: PropsType): ReactElement => {
+  const { t } = useTranslation('error')
+  return (
+    <>
+      <Caption title={cityModel.name} />
+      <ChooseLanguage>{`${t('notFound.language')} ${t('chooseALanguage')}`}</ChooseLanguage>
+      <LanguageSelector
```

Comment:
Would it make sense not only to disable the not-available languages, but to hide them. Think in the header it really makes sense to disable them, but in a list, which says 'select a option' it is a little uninutitive.

Reply:
Good idea, I think I agree :+1: Other opinions?